salvo/src/lib/api/image.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package salvo;
+
+// Capture all docker images required for the benchmark. This object is examined
+// first before evaluating the state of specified source locations
+message DockerImages {
+  // Determines whether required docker are used if they already exist. The
+  // benchmark image and binary image must be specified. If this is set to false
+  // there must be a specified source location from which we build the image
+
+  // This should be implicit. Images are preferred over building from source. We
+  // build only if the image pull is not successful and sources are present
+  bool reuse_nh_images = 1;
```

Comment:
We seem to spell out `nighthawk` in most names except here. Can we change this name to `reuse_nighthawk_images` instead? It is a bit longer, but longer more explicit names help onboarding and improve code readability.